public/app/features/dashboard-scene/scene/keyboardShortcuts.ts

```diff
@@ -79,6 +80,20 @@ export function setupKeyboardShortcuts(scene: DashboardScene) {
     onTrigger: () => sceneGraph.getTimeRange(scene).onRefresh(),
   });
 
+  // Zoom out
+  keybindings.addBinding({
+    key: 't z',
+    onTrigger: () => {
+      handleZoomOut(scene);
+    },
+  });
```

Comment:
We are still missing the t left arrow and t right arrow shortcuts (to move time window left or right)

Reply:
yup, will add them soon to this PR

Reply:
Those are added now